ui/components/src/LineChart/index.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export * from './LineChart';
+export * from './utils';
```

Comment:
Are all the functions in `utils` meant to be part of the public API? Looked like maybe only some of them were for external use? Just want to make sure we're only exposing what we need to since once an API is public, changing it means publishing a major version update.